spring-cloud-aws-parameter-store-config/pom.xml

```diff
@@ -62,5 +62,17 @@
 			<optional>true</optional>
 		</dependency>
 
+		<dependency>
```

Comment:
I am not sure about bringing these dependencies for the validation

Reply:
They have test scope and since in 2.3 we moved away from validation API, in 2.3 they will be removed